crates/rpc/rpc-types/src/eth/pubsub.rs

```diff
@@ -97,6 +104,8 @@ pub enum Params {
     None,
     /// Log parameters.
     Logs(Box<Filter>),
+    /// New pending transaction parameters.
+    NewPendingTransactions(bool),
```

Comment:
this, we want this at the last positon because should be less common than filter